season-11/venues.csv

```diff
@@ -0,0 +1,19 @@
+FTB,Full Tilt Ballard
```

Comment:
`venues.csv` might not really be useful anymore. For a couple seasons, `venues.json` has been the real source of truth. I will likely delete this.